base_layer/core/src/base_node/comms_interface/local_interface.rs

```diff
@@ -82,22 +83,31 @@ impl LocalNodeCommsInterface {
     }
 
     /// Request the block header of the current tip at the block height
-    pub async fn get_blocks(&mut self, block_heights: Vec<u64>) -> Result<Vec<HistoricalBlock>, CommsInterfaceError> {
+    pub async fn get_blocks(
+        &mut self,
+        start: u64,
```

Comment:
I would much prefer the use of range here to avoid inclusive/exclusive inconsistency in methods
```suggestion
        range: Range<u64>,
```

Reply:
Makes sense, I originally tried to make it generic on `RangeBounds<u64>` but that won't work since `RangeBounds<u64>` cannot be made into a trait object.  Think it makes sense to force the use of `RangeInclusive` syntax e.g.`0..=10`. Will change